packages/atlas/src/MainLayout.tsx

```diff
@@ -80,14 +81,16 @@ export const MainLayout: React.FC = () => {
   return (
     <>
       <CookiePopover />
-      <Routes>
-        <Route path={absoluteRoutes.embedded.video()} element={<EmbeddedView />} />
-        <Route path={BASE_PATHS.viewer + '/*'} element={<ViewerLayout />} />
-        <Route path={BASE_PATHS.legal + '/*'} element={<LegalLayout />} />
-        <Route path={BASE_PATHS.studio + '/*'} element={<LoadableStudioLayout />} />
-        <Route path={BASE_PATHS.playground + '/*'} element={<LoadablePlaygroundLayout />} />
-      </Routes>
-      <AdminOverlay />
+      <ReactionsTransactionsProvider>
```

Comment:
What's the reason to keep this provider here? All providers live in `App.tsx`

Reply:
fixed!